### ISSUE TITLE
Markdown Preview not working

### ISSUE THREAD

**Issue by 4015-al** [VSCodium/vscodium#1512]
**Describe the bug**
The Markdown Preview for an MD file is not displayed when I open it using 

**Please confirm that this problem is VSCodium-specific**
- [x] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.

Tested in the same  environment with
-  Visual Studio Code - Version 1.78.1
-  VSCodium - Version 1.78.1

**Please confirm that the issue/resolution isn't already documented**
- [x] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/DOCS.md) and my issue is not mentioned there.

**To Reproduce**
1. Create a markdown file: 'test.MD'

2. Fill it with any valid markdown: 
```md
# Hello, World!
can you hear me?
```
3. while opening the MD file, use any option that is supposed to open its preview, such as `Ctrl+K V` or  `Ctrl+Shift+V` or the GUI by clicking on Preview button.

4. See error: a blank preview! 

**Expected behavior**
a rendered preview of the MD file 

**Screenshots**

- Current Behavior (VSCodium)
![vscodium](https://github.com/VSCodium/vscodium/assets/91027619/42ec62ac-31ec-4991-8fa4-8507781d27f9)

- Expected Behavior (Visual Studio Code)
![visual studio code](https://github.com/VSCodium/vscodium/assets/91027619/9c29223d-621d-4660-adbc-37fc47a9f7d2)

**Desktop (please complete the following information):**
```sh
# os
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.2 LTS
Release:	22.04
Codename:	jammy 
# architecture
x86_64
```
**Additional context**
- I have disabled all the markdown related add-on for both editors, afaik
- I think this issue is related to the last update
- VSCodium is installed by downing the official release directly from https://vscodium.com/
- Visual Studio Code is installed recently using `apt install`

**Comment by tuupola** [comment]
Same problem with Fedora 36.

```
$ cat /etc/fedora-release  
Fedora release 36 (Thirty Six)

$ dnf info codium  
Last metadata expiration check: 0:00:07 ago on 2023-05-11T15:48:51 EEST.
Installed Packages
Name         : codium
Version      : 1.78.1.23130
Release      : el7
Architecture : x86_64
Size         : 313 M
Source       : codium-1.78.1.23130-el7.src.rpm
Repository   : @System
From repo    : gitlab.com_paulcarroty_vscodium_repo
```

**Comment by tuupola** [comment]
Most likely duplicate of https://github.com/VSCodium/vscodium/issues/1503

**Comment by daiyam** [comment]
@4015-al the fixed release is `1.78.1.23131` (still building).

I did close your issue because it was a duplicate :wink:
